src/main/java/com/coveros/selenified/element/Get.java

```diff
@@ -424,8 +424,7 @@ public int numOfTableColumns() {
         }
         WebElement thisRow = rows.get(rowNum);
         List<WebElement> cells = thisRow.findElements(By.xpath(".//th|.//td"));
-        List<WebElement> row = new ArrayList<>();
-        row.addAll(cells);
+        List<WebElement> row = new ArrayList<>(cells);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "row". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1488)